_pages/Source-code-and-configuration-management.md

```diff
@@ -1,5 +1,82 @@
 ###Source Code and Configuration Management
 
+####Commenting
+
+All methods should begin with a brief comment describing the functional characteristics of the method (what it does). This description should not describe the implementation details (how it does it) because these often change over time, resulting in unnecessary comment maintenance work, or worse yet erroneous comments. The code itself and any necessary inline or local comments will follow the standards as defined by the [SalesforceFoundation](https://github.com/SalesforceFoundation/ApexDoc#documenting-class-files):
+
+#####Class Comments
+
+Located in the lines above the class declaration. The special tokens are all optional.
+
+|token | description|
+|-------------|:--------|
+|@author | the author of the class|
+|@date | the date the class was first implemented|
+|@group | a group to display this class under, in the menu hierarchy|
+|@group-content | a relative path to a static html file that provides content about the group|
+|@description | one or more lines that provide an overview of the class|
+
+Example
+
+```
+/**
+* @author Salesforce.com Foundation
```

Comment:
TRIVIAL: The codebase is tending toward aligning the first asterisks as is the style on [jsdoc](http://usejsdoc.org/about-getting-started.html) so that Apex and JS commenting are more congruent.

```
/**
 * This is an example.
 * @example
 */
```

vs.

```
/**
* This is an example.
* @example
*/
```

The doc generation tool won't care though, so this it totally stylistic.